src/components/Grid/Container.js

```diff
@@ -33,6 +23,29 @@ Container.propTypes = {
    * class name
    */
   className: PropTypes.string,
+  /**
+   * flavor: string or array of strings. Flavors: small, medium, large, x-large,
+   * fluid, left, center, right
+   */
+  flavor: PropTypes.oneOfType([PropTypes.oneOf([
+    'small',
```

Comment:
I really think we should move sizes to a dedicated prop

Reply:
👌